tutorials/hyperspectral/click_points_rough.R

```diff
@@ -0,0 +1,70 @@
+
```

Comment:
I think these must be scrips where you're testing out items?  We don't want to pull these into the main repo, I don't think.  Can they be moved to a branch or just stored locally?